mmdet/apis/ote/sample/sample.py

```diff
@@ -146,14 +144,14 @@ def main(args):
         logger.info(str(performance))
 
         logger.info('Run POT optimization')
-        optimized_model = OptimizedModel(
+        optimized_model = Model(
             NullProject(),
             NullModelStorage(),
             dataset,
             environment.get_model_configuration(),
-            ModelOptimizationType.POT,
+            optimization_type=ModelOptimizationType.POT,
             optimization_methods=[],
```

Comment:
Is there a suitable option for optimization method to put here?

Reply:
As far as I understand, there are NNCF optimization methods only, not POT

Reply:
It may be, but it's not stated anywhere explicitly. So, you feel that 
`OptimizationMethod.QUANTIZATION` is not a valid option here, right?

Reply:
added